src/test/resources/docker/docker-compose.yml

```diff
@@ -2,32 +2,42 @@ version: '3.3'
 
 services:
 
-  mysql-5.7:
-    image: library/mysql:5.7.31
-    ports:
-      - "33062:3306"
-    restart: always
-    environment:
-      MYSQL_ROOT_PASSWORD: LbRootPass1
-      MYSQL_DATABASE: lbcat
-      MYSQL_USER: lbuser
-      MYSQL_PASSWORD: LiquibasePass1
-    volumes:
-      - "./mysql-init.sql:/docker-entrypoint-initdb.d/mysql-init.sql"
+#  mysql-5.7:
```

Comment:
@r2liquibase -- please add back (uncomment) all the other dbs that you have commented out in this file.